bitnami/redis-cluster/values-production.yaml

```
@@ -280,6 +280,11 @@ initJob:
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ##
   affinity: {}
+
+  ## Priority class name
+  ##
+  # priorityClassName: ""
```

Does it make sense to uncomment the parameter and leave it as nil? The same than `redis.priorityClassName`:

```suggestion
  priorityClassName:
```